src/upload.rs

```diff
@@ -200,6 +200,47 @@ impl<'a> Uploader<'a> {
         result
     }
 
+    /// Wrapper: Delete the given file from the server.
+    pub fn delete_file(&self, file: &'a str) -> UploadResult<'a, String> {
+        UploadResult(file, self.delete(file))
+    }
+
+    /// Delete the given file from the server.
+    fn delete(&self, file: &'a str) -> Result<String> {
+        let url = self.retrieve_url(file)?;
+        let mut request = self.client.delete(url.as_str());
+        if let Some(delete_token) = &self.config.server.delete_token {
+            request = request.set("Authorization", delete_token);
+        }
+        let result = match request.call() {
+            Ok(response) => {
+                let status = response.status();
+                if status == 200 {
+                    Ok("Deleted".to_string())
```

Comment:
I think we can just print out the server response here. (see https://github.com/orhun/rustypaste/pull/136/commits/6fd3e1993f16eccc0f487e4078734baa9cf18dd8)

Reply:
Yes, I can do that, even though I think that `Deleted` is more than enough. When I wrote the code, the server did not return a message on success, which was fine, since it returned status 200. (First I also thought about adding a message on the server on success to `file deleted successfully`, but decided against it.)

In either case, can we change the current message on the server?

```
the file is deleted
```

should rather be one of the following:

```
file has been deleted
file deleted successfully
```

An article is not used in any other message at the beginning. `has been` is in this context more correct.

If you are ok with changing the message, please let me know which one you prefer and I submit a PR server side.

In the meantime I'll change the code here to return the server response message.